### PR TITLE
BGDIINF_SB-2503 : matching old viewer style for textual buttons in drawing mode

### PR DESCRIPTION
**Pull request by ltkum**
[Test link](https://sys-map.dev.bgdi.ch/feature_bgdiinf-sb-2503-match-old-style-for-buttons/index.html)

### CODE REVIEW

File: src/modules/drawing/components/DrawingExporter.vue [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <template>
     <DropdownButton
-        class="m-1"
+        class="m-1 drawing-button border-right"
```
Comment (jedef) [inline src/modules/drawing/components/DrawingExporter.vue:3]: As these classes are defined in the parent module (and also scoped only to the parent module), I think you should also assign them in the parent class, as else it breaks the independence of the modules and is harder to understand. I.e. like this:
`<DrawingExporter :is-drawing-empty="isDrawingEmpty" class="drawing-button border-right" />`

File: src/modules/drawing/components/DrawingToolbox.vue [hunk 2]
```diff
@@ -206,6 +209,21 @@ $zindex-drawing-toolbox: -1;
...
 }
+.drawing-button {
+    touch-action: manipulation;
+    cursor: pointer;
+    background-image: none;
+    background-color: transparent;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    width: 33%;
+}
```
Comment (jedef) [inline src/modules/drawing/components/DrawingToolbox.vue:220]: Lots of the properties in the "drawing-button" class (like background-image and background-color) seem to be the default values. Maybe you could remove the attributes that are not necessary.

**Review by jedef — COMMENTED** [review]
image

The dropdown menu of the drawing exporter is placed on the separator. Could you please fix that?

File: src/modules/drawing/components/DrawingExporter.vue (hunk 1 above)
Comment (ltkum) [reply src/modules/drawing/components/DrawingExporter.vue]: This seems like a good idea. It is now done.

File: src/modules/drawing/components/DrawingToolbox.vue (hunk 2 above)
Comment (ltkum) [reply src/modules/drawing/components/DrawingToolbox.vue:223]: I removed what was necessary to remove. I left the background-color as it is not transparent by default and we need it to be transparent.

**Review by jedef — COMMENTED** [review]
image

Sorry if I am nitpicking, but now it seems that the englisch version is not that beautiful, as the spacing seems uneven. As you seem to be in a flex container, you can probably achieve what you want by setting 'flex: 1' on the buttons.

Also with the button on the left, hovering over it changes the color of the text, while hovering over the two other buttons changes their background color, which is a bit inconsistent. So I would change it that hovering the left button also changes the background color and not the text color.

**Comment by pakb** [comment]
The "export" button still has some border around it (and between the caret and button), is it possible to easily remove it?

**Comment by pakb** [comment]
As discussed in our "Button talk" we will close this one, and switch the button to the default bootstrap `btn-light` instead of replicating mf-geoadmin3's code

**Pull request closed without merging**